torch/linalg/__init__.py

```diff
@@ -378,6 +378,73 @@
             [1, 2, 2, 2]])
 """)
 
+vector_norm = _add_docstr(_linalg.linalg_vector_norm, r"""
+linalg.vector_norm(input, ord=None, keepdim=False, *, out=None, dtype=None) -> Tensor
+
+Returns the vector norm of a given tensor. 
+
+Args:
+    input (Tensor): The input tensor. If the input has more than one dimension,
+        it is interpretted as a tensor flattened to one dimension.  The input
+        data type must be either a floating point or complex type. For complex
+        inputs, the norm is calculated on of the absolute values of each
```

Comment:
"is calculated on of the absolute values of each element" -> "is calculated using the absolute value of each element"